plugins/Installation/lang/en.json

```diff
@@ -137,6 +137,8 @@
         "SystemCheckPhpSetting": "To prevent some critical issue, you must set the following in your php.ini file: %s",
         "SystemCheckUpdateHttps": "Update over HTTPS",
         "SystemCheckUpdateHttpsNotSupported": "Matomo cannot use HTTPS to update, it will fall back to the insecure HTTP update. Check that CURL or allow_url_fopen is supported and that the openssl PHP extension is installed: https:\/\/matomo.org\/faq\/troubleshooting\/faq_177\/.",
+        "PhpBinaryCheck" : "64-bit PHP Binary",
+        "PhpBinaryCheckHelp": "32-bit <p> We highly recommend upgrading to a 64-bit PHP binary to prevent various issues that may cause inconvenience in the future.</p>",
```

Comment:
FYI I've done a 3 minute check to see if there are already any issues re integer overflows already and couldn't find any except for the ignore cookie (see issue https://github.com/matomo-org/matomo/issues/18019). Searched for eg `time() +` and `86400` and `365` in the code base.  It actually doesn't look like there will be any issues for the next 5-10 years but this can obviously change any time. 

Maybe could say 

```
We recommend upgrading to a 64-bit PHP binary to prevent bugs in the future.
```

I don't think there's a big urgency just yet which is why I would maybe remove the word "highly" recommended. and instead of "inconvenience" would say maybe something like "bugs" to make it more clear.
Or wondering if we maybe even need to show this yet at all assuming there won't be an issue for the next few years maybe? Meaning we could maybe better say `We recommend upgrading to a 64-bit PHP binary at the latest by January 2026 to prevent bugs in the future.`

When we say "in the future" then it might not be clear for the user when there will be a problem and they might panic and stop using Matomo if it's not easy to switch to a 64 bit right now.

Unless there are maybe some other integer overflows in there already? I'm assuming they might be hard to find?